reader.go

```diff
@@ -443,15 +443,20 @@ func decodeLineOfMediaPlaylist(p *MediaPlaylist, wv *WV, state *decodingState, l
 				return fmt.Errorf("Byterange sub-range offset value parsing error: %s", err)
 			}
 		}
-	case !state.tagScte35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
-		state.tagScte35 = true
+	case !state.tagSCTE35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
+		state.tagSCTE35 = true
 		state.listType = MEDIA
-		scteData := strings.Split(line[12:], ",")
-		for _, sep := range scteData {
+		SCTEData := strings.Split(line[12:], ",")
+		state.scte = new(SCTE)
+		for _, sep := range SCTEData {
```

Comment:
Can this for loop use the `decodeParamsLine`, similar to the EXT-X-KEY on line 396?